templates/bundles/FOSUserBundle/ChangePassword/change_password.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends "layout.html.twig" %}
```

Comment:
Consider (re)using the FOSUserBundle templates from https://github.com/itk-dev/user-management-bundle/tree/master/src/Resources/views/bundles/FOSUser.

Maybe we should even move the templates to a new bundle (with no dependency on EasyAdmin) to prevent ourselves from copying these template into a lot of projects.